app/src/main/java/fr/free/nrw/commons/feedback/FeedbackContentCreator.java

```diff
@@ -32,11 +35,19 @@ public void init() {
         /*
          * Construct the feedback section title
          */
+
+        //Get the UTC Date and Time and add it to the Title
+        final SimpleDateFormat dateFormat = new SimpleDateFormat("dd/MM/yyyy", Locale.ENGLISH);
```

Comment:
1. Would you mind making it `yyyy/MM/dd` as it is a less ambiguous format?

2. Please append `HH:mm:ss` as users often report several things on the same day, and having the same title breaks the wikilinks.

So to summarize I think `yyyy/MM/dd HH:mm:ss` would be the best.